pkg/controllers/membercluster/v1beta1/membercluster_controller_integration_test.go

```diff
@@ -842,4 +845,215 @@ var _ = Describe("Test MemberCluster Controller", func() {
 			Expect(ns.DeletionTimestamp != nil).Should(BeTrue())
 		})
 	})
+
+	Context("Test membercluster controller force delete with enabling networking agents", func() {
+		BeforeEach(func() {
```

Comment:
most of them are very similiar to the other contexts, any way not to copy paste so many code?